### PR TITLE
Merge OpenTripPlanner and otp-deploy repos

### PR DESCRIPTION
**Pull request by lboyarsky**
Ticket [Merge otp-deploy and OpenTripPlanner repos](https://app.asana.com/0/810933294009540/1131342465209812/f)

### CODE REVIEW

**Comment by lboyarsky** [comment]
@paulswartz good for review now, sorry for adding you too early. 
- Successful build: https://semaphoreci.com/mbta/opentripplanner/branches/lev-merge-with-otpdeploy/builds/22
- Successful deploy: https://semaphoreci.com/mbta/opentripplanner/servers/otp-dev/deploys/17 - the error in the integration tests is the same as the one I mentioned earlier today in https://github.com/mbta/OpenTripPlanner/pull/25. I believe it's ok. 

The main issue was that I've forgotten to un-ignore json config files and apparently in this case OTP build process is using some default settings instead of failing w/ error messages. These default settings produce the graph, which is twice in size comparing to that we usually have and this was the main reason for out-of-memory failures.

File: var/graphs/mbta/router-config.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "type": "real-time-alerts-enhanced",
+      "frequencySec": 100,
+      "url": "https://cdn.mbta.com/realtime/Alerts_enhanced.json",
+      "feedId": "1"
+    },
+    {
+      "type": "stop-time-updater",
+      "frequencySec": 1000,
+      "sourceType": "gtfs-http",
+      "url": "https://cdn.mbta.com/realtime/TripUpdates!.pb",
```
Comment (paulswartz) [inline var/graphs/mbta/router-config.json]: no `!`

File: var/graphs/mbta/router-config.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "updaters": [
+    {
+      "type": "real-time-alerts-enhanced",
+      "frequencySec": 100,
+      "url": "https://cdn.mbta.com/realtime/Alerts_enhanced.json",
+      "feedId": "1"
+    },
+    {
+      "type": "stop-time-updater",
+      "frequencySec": 1000,
```
Comment (paulswartz) [inline var/graphs/mbta/router-config.json]: Is this updated every 16 minutes?

File: var/graphs/mbta/router-config.json (hunk 1 above)
Comment (lboyarsky) [reply var/graphs/mbta/router-config.json]: Sorry, my bad, this is the config I was using for testing. Fixed.

File: var/graphs/mbta/router-config.json (hunk 2 above)
Comment (lboyarsky) [reply var/graphs/mbta/router-config.json]: Same as above, should be 10 seconds, fixed.

**Pull request merged**